apps/comps/components/user-vote.tsx

```diff
@@ -30,10 +30,12 @@ export const UserVote: React.FC<UserVoteProps> = ({
   competitionId,
   totalVotes,
 }) => {
-  const { data: agentData } = useAgent(agentId);
-  const { data: competitionAgentsData } = useCompetitionAgents(competitionId, {
-    filter: agentData?.agent.name,
-  });
+  // TODO: Filter by agent name once the api supports it.
+  const { data: competitionAgentsData } = useQuery(
+    tanstackClient.competitions.getAgents.queryOptions({
+      input: { competitionId },
+    }),
+  );
```

Comment:
The filter param provided by the old code wasn't actually used by the service, so deleted it. We're going to delete this file anyways since we no longer do voting.